package.json

```diff
@@ -39,7 +40,8 @@
     "react-dom": "^17.0.1",
     "react-helmet": "^6.1.0",
     "react-minimal-pie-chart": "^8.2.0",
-    "stream-browserify": "^3.0.0"
+    "stream-browserify": "^3.0.0",
+    "uuid": "^8.3.2"
```

Comment:
- Let's see if we can remove UUID dependency, the less packages we use the better

Reply:
Yes, i will use the contract address  as ID